src/containerapp/azext_containerapp/_params.py

```diff
@@ -440,3 +440,13 @@ def load_arguments(self, _):
         c.argument('logger_name', help="The logger name.")
         c.argument('logger_level', arg_type=get_enum_type(["off", "error", "info", "debug", "trace", "warn"]), help="Set the log level for the specific logger name.")
         c.argument('all', help="The flag to indicate all logger settings.", action="store_true")
+
+    with self.argument_context('containerapp debug') as c:
+        c.argument('container',
+                   help="The container name that the debug console will connect to. Defaults to the first container of first replica.")
```

Comment:
```suggestion
                   help="The container name that the debug console will connect to. Defaults to the first container of first container.")
```

Reply:
first container of first replica, this is correct.